[2025-03-02T12:00:03.760+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-02T12:00:03.766+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: forecast_stock_price.predict_forecast manual__2025-03-02T11:58:19.192262+00:00 [queued]>
[2025-03-02T12:00:03.768+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: forecast_stock_price.predict_forecast manual__2025-03-02T11:58:19.192262+00:00 [queued]>
[2025-03-02T12:00:03.768+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-02T12:00:03.772+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): predict_forecast> on 2025-03-02 11:58:19.192262+00:00
[2025-03-02T12:00:03.777+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'forecast_stock_price', 'predict_forecast', 'manual__2025-03-02T11:58:19.192262+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/forecast_stock_price.py', '--cfg-path', '/tmp/tmp836tyst4']
[2025-03-02T12:00:03.778+0000] {standard_task_runner.py:105} INFO - Job 38: Subtask predict_forecast
[2025-03-02T12:00:03.778+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=363) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-02T12:00:03.779+0000] {standard_task_runner.py:72} INFO - Started process 366 to run task
[2025-03-02T12:00:03.806+0000] {task_command.py:467} INFO - Running <TaskInstance: forecast_stock_price.predict_forecast manual__2025-03-02T11:58:19.192262+00:00 [running]> on host 1d1d19bba66e
[2025-03-02T12:00:03.846+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='forecast_stock_price' AIRFLOW_CTX_TASK_ID='predict_forecast' AIRFLOW_CTX_EXECUTION_DATE='2025-03-02T11:58:19.192262+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-02T11:58:19.192262+00:00'
[2025-03-02T12:00:03.847+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-02T12:00:03.871+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-03-02T12:00:03.871+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-02T12:00:03.872+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-02T12:00:26.465+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T12:00:27.155+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T12:00:27.157+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-02T12:00:27.173+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-02T12:00:27.174+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=forecast_stock_price, task_id=predict_forecast, run_id=manual__2025-03-02T11:58:19.192262+00:00, execution_date=20250302T115819, start_date=20250302T120003, end_date=20250302T120027
[2025-03-02T12:00:27.228+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-02T12:00:27.241+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-02T12:00:27.242+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
